CHANGELOG.md

```diff
@@ -45,6 +51,8 @@
 - [BREAKING] Normalized naming of `verify` procedures of ECDSA precompile ([#2413](https://github.com/0xMiden/miden-vm/issues/2413)).
 - Refactored Blake3_256 fingerprints to allocate less ([#2375](https://github.com/0xMiden/miden-vm/pull/2375)).
 - [BREAKING] Normalized signature encoding methods in the `dsa` module of the core library.
+- [BREAKING] Make the Assembler work in debug mode, remove optionality ([#2396](https://github.com/0xMiden/miden-vm/pull/2396)).
+- [BREAKING] Normalize naming of `verify` procedures of ECDSA precompile ([#2413](https://github.com/0xMiden/miden-vm/issues/2413)).
```

Comment:
Those modify the 0.20 changelog; do they belong there still?

Reply:
Ah, yeah, I need to polish that!